CHANGELOG.md

```diff
@@ -2,6 +2,31 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [3.4.0](https://github.com/sonata-project/ecommerce/compare/3.3.0...3.4.0) - 2020-10-19
+### Added
+- [[#688](https://github.com/sonata-project/ecommerce/pull/688)] Support for "friendsofsymfony/rest-bundle:^3.0" ([@wbloszyk](https://github.com/wbloszyk))
+- [[#685](https://github.com/sonata-project/ecommerce/pull/685)] Added public alias `Sonata\BasketBundle\Controller\Api\BasketController` for `sonata.basket.controller.api.basket` service ([@wbloszyk](https://github.com/wbloszyk))
+- [[#685](https://github.com/sonata-project/ecommerce/pull/685)] Added public alias `Sonata\CustomerBundle\Controller\Api\AddressController` for `sonata.customer.controller.api.address` service ([@wbloszyk](https://github.com/wbloszyk))
+- [[#685](https://github.com/sonata-project/ecommerce/pull/685)] Added public alias `Sonata\CustomerBundle\Controller\Api\CustomerController` for `sonata.customer.controller.api.customer` service ([@wbloszyk](https://github.com/wbloszyk))
+- [[#685](https://github.com/sonata-project/ecommerce/pull/685)] Added public alias `Sonata\InvoiceBundle\Controller\Api\InvoiceController` for `sonata.invoice.controller.api.invoice` service ([@wbloszyk](https://github.com/wbloszyk))
+- [[#685](https://github.com/sonata-project/ecommerce/pull/685)] Added public alias `Sonata\OrderBundle\Controller\Api\OrderController` for `sonata.order.controller.api.order` service ([@wbloszyk](https://github.com/wbloszyk))
+- [[#685](https://github.com/sonata-project/ecommerce/pull/685)] Added public alias `Sonata\ProductBundle\Controller\Api\ProductController` for `sonata.product.controller.api.product` service ([@wbloszyk](https://github.com/wbloszyk))
+
+### Changed
+- [[#688](https://github.com/sonata-project/ecommerce/pull/688)] Support for deprecated "rest" routing type in favor for xml ([@wbloszyk](https://github.com/wbloszyk))
+- [[#687](https://github.com/sonata-project/ecommerce/pull/687)] Translate flash message before add it to `flash bag` ([@wbloszyk](https://github.com/wbloszyk))
+
+### Fixed
+- [[#685](https://github.com/sonata-project/ecommerce/pull/685)] Fix RestFul API - `Class could not be determined for Controller identified` Error ([@wbloszyk](https://github.com/wbloszyk))
```

Comment:
The referenced error message seems inconclusive.

Reply:
I just used dev-kit to auto-generate this. If you prefer something else give a suggestion.
You can also push to this branch.

I don't know if we can do something when merging PR, because making a new release shouldn't take time to rewrite changelog messages IMHO. Does a ci exist could exist to check the PR changelog before merging it ?

Reply:
I agree that we should change the messages as less as possible. I'm just pointing the concerns I have watching these messages.

Reply:
BTW, regarding the error message, I don't know what the original message was.